### PR TITLE
tugboat config file

### PR DESCRIPTION
**Pull request by rmessina1010**
## Description
Testing tugboatQA

### CODE REVIEW

**Comment by rmessina1010** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr22376-e8fwu2ti6w00krn31ycs6bjspb7dsdsx.tugboat.vfs.va.gov


Dashboard:
- https://tugboat.vfs.va.gov/635ab95c4b29f55af65ca56e

File: .tugboat/config.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+services:
+  node:
+    image: tugboatqa/node:14
+    default: true
+    expose: 3001
+    commands:
```
Comment (q0rban) [inline .tugboat/config.yml:6]:
```suggestion
    commands:
      init:
        - mkdir -p /etc/service/node
        - echo "#!/bin/sh" > /etc/service/node/run
        - echo "npm --prefix=$TUGBOAT_ROOT run heroku-serve -- build/localhost -p 3001" >> /etc/service/node/run
        - chmod +x /etc/service/node/run
```

File: .tugboat/config.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+      build:
+        - yarn
+        - yarn build
```
Comment (q0rban) [inline .tugboat/config.yml:9]:
```suggestion
        - yarn build --host=0.0.0.0 --public=$TUGBOAT_DEFAULT_SERVICE_URL_HOST
```

File: .tugboat/config.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+      start:
```
Comment (q0rban) [inline .tugboat/config.yml:10]:
```suggestion
```

File: .tugboat/config.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+        - yarn watch &
```
Comment (q0rban) [inline .tugboat/config.yml:11]:
```suggestion
```

File: .tugboat/config.yml (hunk 1 above)
Comment (q0rban) [reply .tugboat/config.yml:6]: This will add the npm command that starts the http-server into a runit script, so that it will run in the background

**Comment by rmessina1010** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr22376-e8fwu2ti6w00krn31ycs6bjspb7dsdsx.tugboat.vfs.va.gov


Dashboard:
- https://tugboat.vfs.va.gov/635ab95c4b29f55af65ca56e

**Comment by ElijahLynn** [comment]
I triggered this rebuild because we had an issue with our Tugboat server and it needed to be rebuilt, I know this is not needed anymore though but just wanted get that environment working like all the other environments, looks like @rmessina1010 is still setup as the user.

**Pull request closed without merging**